components/infobox/wikis/starcraft2/infobox_building_custom.lua

```diff
@@ -155,22 +155,21 @@ function CustomBuilding._getRace(race)
 end
 
 function CustomBuilding:_getCostDisplay()
-	local minerals = _args.min or 0
-	---@diagnostic disable-next-line: cast-local-type
-	minerals = ICON_MINERALS .. '&nbsp;' .. minerals
+	local mineralValue = _args.min or 0
+	local minerals = ICON_MINERALS .. '&nbsp;' .. mineralValue
 
 	local gas = _args.gas or 0
 	gas = (ICON_GAS[_race] or ICON_GAS['default']) .. '&nbsp;' .. gas
```

Comment:
Should gas nee the same tweak as minerals?

Reply:
looking at this can probably make it
```lua
	local minerals = ICON_MINERALS .. '&nbsp;' .. (tonumber(_args.min) or 0)

	local gas = (ICON_GAS[_race] or ICON_GAS['default']) .. '&nbsp;' .. (tonumber(_args.gas) or 0)
```